src/PerformanceTests/NServiceBus6/Session.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+
+class Session : ISession
+{
+    readonly IEndpointInstance instance;
+
+    public Session(IEndpointInstance instance)
+    {
+        this.instance = instance;
+    }
+
+    public Task Send(object message)
+    {
+        return instance.Send(message);
```

Comment:
@danielmarbach This is allowed correct? It is not required to do:

```c#
public async Task Send(object message)
{
    await instance.Send(message);
}
```

It feels strange to await when you can just return but I don't know if this is bad practice and if it is, why its so.